### DISCUSSION TITLE
(lazer) abilty to delete maps / map elements in bulk

### DISCUSSION THREAD

**Discussion by 892kao196** [ppy/osu#31132] [Ideas]
allow to delete all beatmap backgrounds / video / hitsounds in lazer to save space. make the process of bulk deleting maps simple (as well as bulk deleting maps that have a low star rating for example)

**Comment by Joehuu** [comment]
There's already an option to delete all videos in settings (rationale for this existing is probably because you can download without video already).

As for the other elements, probably not happening (at least for downloads), see https://github.com/ppy/osu/discussions/18404#discussioncomment-2816231.
